### PR TITLE
TRUNK-4522: Fix HttpClientTest test on JVM 1.8.

### PR DESCRIPTION
**Pull request by approce**
Fix test post_shouldPostUrlParametersAndGetResponse(org.openmrs.util.HttpClientTest)

### CODE REVIEW

File: api/src/test/java/org/openmrs/util/HttpClientTest.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public void post_shouldPostUrlParametersAndGetResponse() throws Exception {
 		verify(connection).setRequestProperty("Content-Length", String.valueOf(16));
 		verify(connection).setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
 		
-		assertThat(stream.toString(), is("&two=two&one=one"));
+		assertThat(stream.toString(), is("&one=one&two=two"));
```
Comment (approce) [inline api/src/test/java/org/openmrs/util/HttpClientTest.java]: In master branch this fix already exist.

**Comment by dkayiwa** [comment]
@approce did you just forget to claim this ticket? https://issues.openmrs.org/browse/TRUNK-4522

**Comment by dkayiwa** [comment]
@approce can you also always include at least the ticket summary in the pull request message as advised at http://go.openmrs.org/pull-request-tips?

**Comment by approce** [comment]
@dkayiwa yeap, I will always include ticket summary to PR :)

**Pull request merged**